### ISSUE TITLE
We need a new release desperately

### ISSUE THREAD

**Issue by smac89** [jupyterhub/systemdspawner#93]
### Proposed change
I've been scratching my head for almost a day now wondering why `{USERNAME}` and `{USERID}` are not expanded in extra properties. I kept looking at the code in this repo, and tracing all the steps and everything suggests that those variables should be expanded, yet when I run my jupyterhub, after the first user is created, any subsequent user receives 500 server error because the runtime directory created is not expanded, and the unit attempts to use the directory of another.

I see that the last release was in 2020, but the [commit](https://github.com/jupyterhub/systemdspawner/commit/e75caddd7f310c591f642cdd559107c171a1e657) that allows expanding the variables was done in 2021.

### Alternative options
Make a partial release which only includes this one commit

### Who would use this feature?
Anyone who has read the readme recently
![ArcoLinux_2022-04-21_21-05-52](https://user-images.githubusercontent.com/8305511/164588496-8764af40-1327-446d-a404-76f2feef7b92.png)

**Comment by smac89** [comment]
If it helps, this is how I use `unit_extra_properties`:

```
c.SystemdSpawner.unit_extra_properties = {
    "RuntimeDirectory": os.path.join(
        *(
            os.getenv("RUNTIME_DIRECTORY")
            .split(os.path.pathsep)[0]
            .split(os.path.sep)[2:]
        ),
        '{USERNAME}',
        # TODO: Would be nice to use {USERID} or {USERNAME} here...
    ),
    ...
}
```

**Comment by yuvipanda** [comment]
Thanks for the ping, @smac89! I've just released v0.16 with that PR included. Apologies for the delay, and sorry about the frustration it caused. I'm glad you were able to find a workaround though!
